creator-node/src/app.js

```diff
@@ -199,6 +202,10 @@ const initializeApp = (port, serviceRegistry) => {
   // https://expressjs.com/en/guide/behind-proxies.html
   app.set('trust proxy', true)
 
+  // Initialize premium content access checker and make accessible via app handle
+  const premiumContentAccessChecker = new PremiumContentAccessChecker()
+  app.set('premiumContentAccessChecker', premiumContentAccessChecker)
+
```

Comment:
I do agree a fn is simpler, but i want a class that i can stub for the other tests that have depend on it.
I did remove the interface though, although I dont think it hurt and it would only enforce that the functions would follow the same signature... but removed it for more simplicity